### ISSUE TITLE
No loaders are configured to process this file

### ISSUE THREAD

**Issue by xvThomas** [AlexJWayne/ts-gl-shader#1]
Hello Alex, good job!

I got the following error on:
`const shader = createShaderProgram(gl, vertSrc, fragSrc)`

```
Error: Module parse failed: Unexpected token (52:56)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|       identifier
|     } = declaration;
>     const attributeSize = Number(type.match(/([234])$/)?.[1]) || 1;
|     const location = gl.getAttribLocation(program, identifier);
|     handleGlError(gl, `ShaderProgramObject gl.getAttribLocation() ${identifier}`);
    at ./node_modules/@alexjwayne/ts-gl-shader/dist/alexjwayne-ts-gl-shader.esm.js
```

My shaders are declared as a const string as you emphasize in your documentation and I do no longer use glsl files. 
I have also reduced them to the minimum:
```
const vertSrc = /* glsl */ `
attribute vec2 aVertexPosition;
attribute vec4 aVertexColor;

void main(void) {
    vec2 position = aVertexPosition;
    vColor = aVertexColor;          
}
` as const

const fragSrc = /* glsl */ `
varying lowp vec4 vColor;
uniform lowp float opacity;

void main(void) {
    gl_FragColor = vColor * opacity;
}
` as const
```

However, my code is running in storybook and I suspect something on that side...
Any idea ?

**Comment by AlexJWayne** [comment]
Thanks for taking a look at the lib!

Looks like webpack is choking on the optional `?.` chaining operator. I think I have too modern of a compilation target set. I'll have an update on this soon.

**Comment by xvThomas** [comment]
Thanks to keep me updated me as soon as possible. Your approach is quite interesting. I plan to embed your lib in mine (on going work, not yet published on GitHub) - I have also other questions regarding Mat4 use, but let's fix this one first.

**Comment by AlexJWayne** [comment]
I just published 0.1.2 which actually respects the compilation target of ES2019.

That line now looks like this in the compiled output:

```js
const attributeSize = Number((_a = type.match(/([234])$/)) == null ? void 0 : _a[1]) || 1;
```

which should play nicer with more babel/webpack configs.

Let me know if that helps!

**Comment by xvThomas** [comment]
Thanks Alex, works fine now !
